plugin-api.d.ts

```diff
@@ -1600,7 +1604,6 @@ interface StickyNode extends OpaqueNodeMixin, MinimalFillsMixin, MinimalBlendMix
   readonly text: TextSublayerNode
   authorVisible: boolean
   authorName: string
-  isWideWidth: boolean
```

Comment:
Sorry for the drive-by review, but I recently added this property. Was removing it an intentional change?